qiskit/algorithms/linear_solvers/hhl.py

```diff
@@ -0,0 +1,373 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2019, 2020.
```

Comment:
These all really ought to have 2021 in them so be dated 2021 or perhaps 2020, 2021 for some since the work started a while back.

Reply:
updated